drake/util/lcmUtil.cpp

```diff
@@ -46,7 +46,8 @@ void verifySubtypeSizes(drake::lcmt_support_data& support_data) {
     throw std::runtime_error("contact_pts must have 3 rows");
   }
   for (int j = 0; j < 3; ++j) {
-    if (support_data.contact_pts[j].size() != support_data.num_contact_pts) {
+    if (static_cast<int32_t>(support_data.contact_pts[j].size()) !=
```

Comment:
Exactly.  Using `int64_t` would be slightly safer to force different promotion-comparison, but it didn't seem worth worrying about.